homeassistant/components/nextdns/__init__.py

```diff
@@ -39,6 +39,22 @@
     NextDnsUpdateCoordinator,
 )
 
+NextDnsConfigEntry = ConfigEntry["NextDnsData"]
+
+
+@dataclass
+class NextDnsData:
+    """Data for the NextDNS integration."""
+
+    connection: NextDnsUpdateCoordinator
+    dnssec: NextDnsUpdateCoordinator
+    encryption: NextDnsUpdateCoordinator
+    ip_versions: NextDnsUpdateCoordinator
+    protocols: NextDnsUpdateCoordinator
+    settings: NextDnsUpdateCoordinator
+    status: NextDnsUpdateCoordinator
```

Comment:
Maybe checkout how I did this with withings, I also have multiple update coordinators